test.py

```diff
@@ -719,7 +760,10 @@ def expectedExec(c):
 start_as_time=time.localtime()
 testRunStartTimeAsEpoch = int(time.time())
 # Need translateModel + make + exe...
-cmd_res=Parallel(n_jobs=n_jobs)(delayed(runScript)(name, 2*data["ulimitOmc"]+data["ulimitExe"]+25, data["ulimitMemory"], verbose) for (model,lib,libName,name,data) in tests)
+if customTimeout > 0.0:
+  cmd_res=Parallel(n_jobs=n_jobs)(delayed(runScript)(name, customTimeout, data["ulimitMemory"], verbose) for (model,lib,libName,name,data) in tests)
+else:
+  cmd_res=Parallel(n_jobs=n_jobs)(delayed(runScript)(name, 2*data["ulimitOmc"]+data["ulimitExe"]+25, data["ulimitMemory"], verbose) for (model,lib,libName,name,data) in tests)
```

Comment:
Instead of duplicating this complicated invocation, why don't you keep it simpler by setting a default and using it appropriately:
```suggestion
if customTimeout == 0:
  customTimeout =  2*data["ulimitOmc"]+data["ulimitExe"]+25  # the default value
cmd_res=Parallel(n_jobs=n_jobs)(delayed(runScript)(name, customTimeout, data["ulimitMemory"], verbose) for (model,lib,libName,name,data) in tests)
```

Reply:
That's right, I'll simplify the code